cirq-google/cirq_google/calibration/phased_fsim_test.py

```diff
@@ -269,6 +269,7 @@ def test_xeb_to_calibration_layer():
     layer_str = str(new_layer)
     # Fix precision issues
     layer_str = re.sub(r'0.004999\d+', '0.005', layer_str)
+    layer_str = re.sub(r'\b0.78539818\d*', '0.7853982', layer_str)
```

Comment:
Could we fix this by setting `theta` on the fsim gate to a different value that can be exactly represented by floats, e.g. `theta=0.5`? We could do something similar on `fatol` to avoid the need for the previous string substitution hack, setting it to something like `2**-7 == 0.0078125` instead of `0.005`.

Reply:
Sure, that is nicer than substitution.  Done in 6148c40eee4c88d27d8c8cfd775f52725deac606.